# message to represent control of the various torso servos. Values are floats
# -1 to 1, where -1 is the minimum extreme, 1 is the maximum extreme, and 0 is
# therefore 0.
float32 left_shoulder
float32 left_elbow
float32 right_shoulder
float32 right_elbow
bool left_finger
bool right_finger

